pkg/job/v1alpha3/consts.go

```diff
@@ -0,0 +1,48 @@
+package v1alpha3
+
+import (
+	"github.com/kubeflow/katib/pkg/controller.v1alpha3/consts"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	// JobNameLabel represents the label key for the job name, the value is job name
+	JobNameLabel = "job-name"
+	// JobRoleLabel represents the label key for the job role, e.g. the value is master
+	JobRoleLabel = "job-role"
+	// TFJobRoleLabel is deprecated in kubeflow 0.7, but we need to be compatible.
+	TFJobRoleLabel = "tf-job-role"
+	// PyTorchJobRoleLabel is deprecated in kubeflow 0.7, but we need to be compatible.
+	PyTorchJobRoleLabel = "pytorch-job-role"
+)
+
+// JobRoleMap is the map which is used to determin if the replica is master.
+// Katib will inject metrics collector into master replica.
+var JobRoleMap = map[string][]string{
+	// Job kind does not support distributed training, thus no master.
+	consts.JobKindJob:     {},
```

Comment:
@gaocegege Just wondering why don't we keep all job related constants in this file?

Reply:
SGTM

Reply:
Does it work for you? @johnugeorge

Reply:
I feel that we have defined too many constants :)